apps/pie-docs/src/components/icon-button/code/code.md

```diff
@@ -124,10 +129,19 @@ import { PieIconButton } from '@justeattakeaway/pie-icon-button';
 
 For React Applications:
 
+```jsx
+import { PieIconButton } from '@justeattakeaway/pie-webc/react/icon-button.js';
+import { IconClose } from '@justeattakeaway/pie-icons-webc';
```

Comment:
Actually I just noticed, I think this should be `/dist/react/IconClose.js` and then the same but without the file extension for React 13